src/pages/integrations/shapeshift/shapeshift-confirm/shapeshift-confirm.ts

```diff
@@ -104,6 +105,9 @@ export class ShapeshiftConfirmPage {
     this.network = this.shapeshiftProvider.getNetwork();
     this.fromWallet = this.profileProvider.getWallet(this.fromWalletId);
     this.toWallet = this.profileProvider.getWallet(this.toWalletId);
+    this.unitToSatoshi = this.configProvider.getCoinOpts()[
+      this.fromWallet.coin.toUpperCase()
```

Comment:
Are you sure this works? It seems that getCoinOpts() returns an object with the different coins in lowercase (not uppercase)

Reply:
removed toUpperCase()